dev/style.css

```diff
@@ -122,3 +122,20 @@ body {
 .ais-show-more__active, .ais-show-more__inactive {
   cursor: pointer;
 }
+
+.ais-header {
+  position: relative;
+}
+
+.collapse-arrow {
+  position: absolute;
+  right: 0;
+}
+
+.collapse-arrow:after {
+  content: "⬆";
```

Comment:
After thinking about it, there's no default "header" template, you always submit yours. In comparison with the item template where we do have a default template.

So I will just pass collapsable and collapsed data to the header template when you use collapsable option.

Along with the "collapsed" CSS class that already exists.